src/components/LoanFinding/QuickFiltersSection.vue

```diff
@@ -28,7 +28,7 @@
 				<template v-for="(loan, index) in loans">
 					<kiva-classic-basic-loan-card-exp
 						v-if="enableLoanCardExp"
-						:key="`new-card-${index}`"
+						:key="`new-card-${loan.id + index}`"
```

Comment:
While it would be super unlikely for the math to cause conflicts, we probably instead want just in case:

```
:key="`new-card-${loan.id}-${index}`"
```